File: junkz/practice_math.py
#!/usr/bin/env python -OB
import math


def main() -> None:
    print(math.cos(135 * math.pi / 180))
    print(math.sqrt(2))
    new_value = 1 / math.sqrt(2)
    print(new_value)
    print(new_value == math.sin(45 * math.pi / 180))


if __name__ == "__main__":
    main()

File: junkz/practice_math.dart
import 'dart:math';

void main() {
  print(sin(45 * pi / 180));
  print(cos(135 * pi / 180));
  print(sqrt(2));
  // exercise
  var new_value = 1 / sqrt(2);
  print(new_value);
  print(new_value == sin(45 * pi / 180));
}
